perfdash/config.go

```diff
@@ -34,9 +34,10 @@ import (
 
 // TestDescription contains test name, output file prefix and parser function.
 type TestDescription struct {
-	Name             string
-	OutputFilePrefix string
-	Parser           func(data []byte, buildNumber int, testResult *BuildData)
+	Name                  string
+	OutputFilePrefix      string
+	Parser                func(data []byte, buildNumber int, testResult *BuildData)
+	forceConstantCategory bool
```

Comment:
Please start with "F" (large latter) to be consistent with other fields here.

Reply:
Thx, done